torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -887,7 +888,63 @@ RegisterOperators reg(
          userObj->setSlot(slot, std::move(v));
          return 0;
        };
-     })});
+     })
+     });
+
+RegisterOperators logging_operators({
+    Operator("prim::BumpCounter(str key, int val) -> ()", [](Stack& stack) {
+          auto val = pop(stack).toInt();
+          auto key = pop(stack).toString();
+
+          auto schema = parseSchema("prim::BumpCounter(str key, int val) -> ()");
```

Comment:
Why is this called prim::BumpCounter when the C++ function is called addStatValue? Can this be made consistent?

Reply:
Because I didn't want to have to rebuild the world after changing interned_strings.h :p let me change it now